src/USBHost3GModule/WANDongleSerialPort.h

```diff
@@ -81,7 +81,7 @@ class WANDongleSerialPort : public IUSBHostSerial {
     *
     * @returns the number of bytes available
     */
-    virtual int writeable();
```

Comment:
Hi @cristidragomir97, Just wanted to point out that I changed the `writeable()` methods to `writable()` across the entire repository. This change shouldn't create any problems, should it?

Reply:
Looks good to me